pySNOM/images.py

```diff
@@ -121,16 +124,16 @@ def transform(self, data):
 
 class LineLevel(Transformation):
 
-    def __init__(self, mtype='median', datatype=DataTypes.Phase):
-        self.mtype = mtype
+    def __init__(self, method='median', datatype=DataTypes.Phase):
```

Comment:
Thanks, these names are nicer. You could go even further and replace `average` with `mean`, because it is more specific.

Reply:
Yes, I agree, I'll do it.